src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -185,6 +196,58 @@ protected int doWork() {
                     " already exists.  Delete this file and try again, or specify a different output file.");
         }
 
+        // map for aliases mapping a contig to its' aliases
+        final Map<String, Set<String>> contig2aliases = new HashMap<>();
+        
+        // fill the alias map
+        if(this.ALT_NAMES != null) {
+            try {
+                // regex defined in the sam spec
+                final Pattern altNameRegex = Pattern.compile("[0-9A-Za-z][0-9A-Za-z\\*\\+@\\|\\-]*");
+                for(final String line :IOUtil.slurpLines(this.ALT_NAMES)) {
```

Comment:
Instead of `IOUtil.slurpLines()` I would use the NIO function `Files.readAllLines()` here.